docs/products/access/topics/custom-registration/custom-registration-v1.md

```diff
@@ -0,0 +1,132 @@
+# Custom Registration v1
+
+## Authentication
+
+These endpoints require HTTP Basic Authentication or url encoded form with the Client Credentials of the Client that 
```

Comment:
Do the URL encoded POST parameters actually work combined with the JSON body? If so, we should mention the parameter names for this.

Reply:
The endpoints expect a JSON body and you can only send a single content-type per request. URL encoded POST parameters are not possible in the same body. I have removed this from the docs.